push_to_docker.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# The image build step occurs only when a build is triggered by a commit being merged into the "master" branch.
+# Given that the "master" branch is protected, the only way to trigger a build from the "master" branch is when a PR
+# is merged into it. This way, we avoid storing images from non-stable branches.
+VERSION=$(cat ./version/version.go | grep "var VERSION" | awk ' { print $4 } ' | sed s/\"//g)
+VERSION_REGEX="^$VERSION$"
+TAG=$(curl https://registry.hub.docker.com/v1/repositories/tfgco/khan/tags | \
+    sed -e 's/[][]//g' -e 's/"//g' -e 's/ //g' | \
+    tr '}' '\n'  | \
+    awk -F: '{print $3}' | \
+    grep "$VERSION_REGEX"
+)
+
+if [ "$TAG" = "$VERSION" ]; then
```

Comment:
Always prefer `[[` to `[`.

Eg: https://unix.stackexchange.com/questions/32210/why-does-parameter-expansion-with-spaces-without-quotes-work-inside-double-brack